MHArmory.Search/MHArmory.Search.csproj

```diff
@@ -4,8 +4,27 @@
     <TargetFramework>netstandard2.0</TargetFramework>
   </PropertyGroup>
 
+  <ItemGroup>
+    <PackageReference Include="Cloo.clSharp" Version="0.9.2.7" />
```

Comment:
Seems not...

`Cloo.clSharp`: https://github.com/clSharp/Cloo
Last commit 6 years ago, no license, no readme. Pending PR since 3 days, not sure what will happen but support for .NET Core is something I'm interested in.

`OpenCL.Net`: https://github.com/aquavit/OpenCL.Net
Last commit 5 years ago. 3 pending PR since 3 years.

Reply:
OpenCL is a native library written in C, it has an official C++ wrapper (not even C++11...) and that's it.  Cloo is as good as you're going to get probably - it's a project by the team that made the OpenTK (Open Toolkit) library, which is by far the most popular OpenGL/OpenAL lib for .NET

If you look at the OpenTK source, very basic OpenCL support was initially included (just P/Invoke wrappers really) but it then got moved out from the project since they are used for very different things - OpenGL/OpenAL for graphics and audio respectively, and OpenCL for GPU computations, and are rarely used together.